modules/server/src/main/scala/scaladex/server/route/api/SearchApi.scala

```diff
@@ -2,32 +2,25 @@ package scaladex.server.route.api
 
 import scala.concurrent.ExecutionContext
 
-import akka.http.scaladsl.model.Uri
-import akka.http.scaladsl.server.Directive
-import akka.http.scaladsl.server.Directive1
 import akka.http.scaladsl.server.Route
 import ch.megard.akka.http.cors.scaladsl.CorsDirectives._
-import com.softwaremill.session.SessionDirectives._
-import com.softwaremill.session.SessionOptions._
```

Comment:
Can we remove the dependency from the build?

Reply:
We still use it on the server side with akka-http